### ISSUE TITLE
[MHV SM / Accessibility] "Provide details" alert does not persist

### ISSUE THREAD

**Issue by sarahhortonadhoc** [department-of-veterans-affairs/va.gov-team#124500]
### 📝 User story

As a user, I want alerts that are relevant to the task at hand to display on the page, so that I can benefit from the guidance provided in the alert as I complete the task.

### 📝 Problem statement

When a user leaves the Start message page to select a different care team and then returns to the Start message page, the alert prompting the user to provide medication details no longer appears on the page.

### 🌌 Supporting screenshots or videos

Top of page includes alert to provide medication details:

<img width="753" height="330" alt="Screenshot of Start message page showing the yellow alert and guidance to add details" src="https://github.com/user-attachments/assets/6a11bab2-50ea-4a8c-b5c1-9e082762becd" />

After changing care team, top of page does not include alert:

<img width="753" height="330" alt="Screenshot of Start message page without yellow alert" src="https://github.com/user-attachments/assets/f55d95d3-8f8a-4480-b216-96b3446beacc" />

### 👣 Reproduction steps

1. Start the renewal task using a non-existent medication, for example, https://staging.va.gov/my-health/secure-messages/new-message/?prescriptionId=2520&redirectPath=%2Fmy-health%2Fmedications (username: karsyn.gpsysasynctwo@id.me)
2. Select a care team and select Continue.
3. Notice the yellow alert on the page: Add your medication information to this message.
4. Select the "Select a different care team" link.
5. Select a care team and select Continue.
6. Notice the yellow alert does not appear on the page.

### 🔗 Point of contact

@sarahhortonadhoc

### ⚠️ Severity level

1, Launchblocker. Critical. Must be fixed before launch.

#### Experience Standards
- [Comprehension: User easily understands the product](https://design.va.gov/about/experience-standards/#category-comprehension)
  - [User has enough information to complete a task](https://design.va.gov/about/experience-standards/#user-has-enough-information-to-complete-a-task)

### 𝌣 Proposed solution or next steps

- Display the alert on the Start message page whenever the page loads without medication details.

### ✅ Acceptance criteria

- [ ] The system displays the alert on the Start message page whenever the page loads without medication details.
- [ ] The user can consult the guidance in the alert whenever they are completing the medication renewal flow for a medication that does not have details.

### ✅ Possible tasks

- [ ] Review, refine, and acknowledge feedback.
- [ ] Update ticket information (title, assignee, labels, project(s), etc.)
- [ ] Fix and/or document decisions made.
- [ ] Product manager to move ticket in their respective product board to 'QA/A11y Review' after reviewing documented decisions / validating fix.
- [ ] Accessibility specialist will close the ticket after reviewing documented decisions / validating the fix.

**Comment by thomsonespy** [comment]
@sarahhortonadhoc I don't think the persistence of this alert is caught in a UX review ticket. This is making me think we should also test the hint text when it is implemented (it is meant to change when there is an error).
